### PR TITLE
feat(WEB-8452): Image Alt support

### PR DESCRIPTION
**Pull request by rcmonsayac**
### What

- https://github.com/martide/editor-js-image/pull/1/files 
- Use martide/editor-js-image dep
- Add custom parser for `image` to use alt attribute 
- Fix alt attr find in `lib/literature/components/image_component.ex`

Note: point to master once https://github.com/martide/editor-js-image/pull/1/files  is merged

### CODE REVIEW

File: assets/package.json
Comment (dgross881) [inline assets/package.json:13]: @rcmonsayac this looks alright but why not open up a PR to the maintainers ?

Comment (rcmonsayac) [reply assets/package.json:13]: @dgross881 it actually is the issue, there is a PR already and I just copied it. Maintainers are not responding here https://github.com/editor-js/image/issues/30 since 2019

Comment (rcmonsayac) [reply assets/package.json:13]: PR in forked repo https://github.com/martide/editor-js-image/pull/1/files, will change this to point to master once merged

**Review by dgross881 — APPROVED** [review]

**Pull request merged**
